### PR TITLE
[ZT] SSO domain requirements

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/applications/configure-apps/dash-sso-apps.md [hunk 1]
```diff
@@ -4,47 +4,47 @@ title: Cloudflare dashboard SSO application
...
 
 {{<Aside type="note">}}
 
-Dashboard SSO is only available to Enterprise customers on the Standard or Premium Success Plans.
+Only available to Enterprise customers on the Standard or Premium Success Plans.
 
 {{</Aside>}}
 
-## Set up dashboard SSO
+By adding a Dashboard SSO application to your Cloudflare Zero Trust account, you can enforce single sign-on (SSO) to the Cloudflare dashboard with the identity provider (IdP) of your choice.
```
Comment (kennyj42) [inline content/cloudflare-one/applications/configure-apps/dash-sso-apps.md]: We should probably also call out that it's a full opt-in for your email domain. If you go SSO, then every user on your email domain **has** to use SSO.

File: content/cloudflare-one/applications/configure-apps/dash-sso-apps.md [hunk 2]
```diff
@@ -4,47 +4,47 @@ title: Cloudflare dashboard SSO application
...
 
-### Step 1 — Launch Cloudflare Zero Trust
+## Prerequisites
 
-To log in to Cloudflare Zero Trust, go to [Zero Trust](https://one.dash.cloudflare.com/).
+Every user that wants to login with Dashboard SSO will need to exist as a member in your Cloudflare account and IdP. To add users to your Cloudflare account, refer to [Manage Cloudflare account access](/fundamentals/account-and-billing/members/).
```
Comment (kennyj42) [inline content/cloudflare-one/applications/configure-apps/dash-sso-apps.md]: Would reword a bit. "All users must exist in your Cloudflare account in order to use Dashboard SSO." or something like that. We may also want to call out the Okta user provisioning option.

Comment (ranbel) [reply content/cloudflare-one/applications/configure-apps/dash-sso-apps.md]: SCIM option will be added as part of PCX-6197.

**Review by kennyj42 — APPROVED** [review]
